src/cli/utils/tests/async-load-config.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { unlinkSync, writeFileSync } from 'fs';
+import { asyncLoadConfig } from '../async-load-config';
+import {
+	asyncDataWithAsyncFunctionAndAsyncParameter,
+	asyncFunctionWithAsyncData,
+	asyncyFunctionWithNonAsyncData,
+} from '../fixtures/asyn-load-config-test-fixture';
+
+describe('async load tests', () => {
+	it('should aysnc load medusa-config with non-async', async () => {
+		const data = asyncyFunctionWithNonAsyncData;
+
+		const pathToConfigFile = `${process.cwd()}/../medusa-config.js`;
+		writeFileSync(pathToConfigFile, data);
+		const configModule = await asyncLoadConfig();
```

Comment:
don't need to create the file as the fixture will be consumed, instead just pass the path to the fixture file config to test

Reply:
it will depend on making the path configurable, as medusa's default behaviour is to only pick up medusa-config.js, I didn't want to alter that.
wdyt ?

Reply:
dones